vm/actor/src/builtin/multisig/mod.rs

```diff
@@ -304,75 +324,98 @@ impl Actor {
         })?
     }
 
-    fn approve_transaction<BS, RT>(rt: &mut RT, tx_id: TxnID) -> Result<(), ActorError>
+    fn approve_transaction<BS, RT>(
```

Comment:
As mentioned above, this fn signature has changed, not sure if we want those changes reflected in this PR but figured I would mention it.